### DISCUSSION TITLE
Is there any paper to be referenced in?

### DISCUSSION THREAD

**Discussion by muazhari** [winedarksea/AutoTS#157] [Q&A]
I want to use this library for academic purposes. But, I seem not found anything to be cited. Is there any legit paper provided that explains this library?

**Comment by winedarksea** [comment]
> I want to use this library for academic purposes. But, I seem not found anything to be cited. Is there any legit paper provided that explains this library?

Unfortunately, no, I have not written a paper yet. There is an article on my blog, Syllepsis that could maybe work.
You're not the first to ask for this, so I may need to get around to writing a paper soon.
